VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -182,30 +186,47 @@ def _dict_to_name_value_array(self, values):
                     })
         return array
 
-    def get_settings_data(self, protector_name, kv_url, kv_id, kek_url, kek_kv_id, kek_algorithm, extra_device_items, disk_util, crypt_mount_config_util):
+    def get_settings_data(self, protector_name, kv_url, kv_id, kek_url, kek_kv_id, kek_algorithm, extra_device_items, disk_util, crypt_mount_config_util, key_store_type):
```

Comment:
Double check all calls to this for the added parameter

Reply:
There do not appear to be any additional tests using this function, double checked